webpack.config.ts

```diff
@@ -0,0 +1,75 @@
+import * as autoprefixer from "autoprefixer";
+import * as path from "path";
+import * as webpack from "webpack";
+import "webpack-dev-server"; // Side-effect import just to get devServer types.
+
+const config: webpack.Configuration = {
+  entry: "./docs/docs.jsx",
+  output: {
+    path: path.resolve(__dirname, "docs"),
+    filename: "bundle.js",
+    publicPath: "/",
+  },
+  resolve: {
+    extensions: [".ts", ".tsx", ".js", ".jsx", ".less"],
+    alias: {
+      src: path.resolve(__dirname, "src"),
+      utils: path.resolve(__dirname, "src/utils"),
+      less: path.resolve(__dirname, "src/less"),
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(eot|otf|woff|svg|ttf|png|jpg)$/,
+        loader: "url-loader",
+      },
+      {
+        test: /\.(less|css)$/,
+        use: [
+          "style-loader",
+          "css-loader",
+          {
+            loader: "postcss-loader",
+            options: {
+              postcssOptions: {
+                plugins: [autoprefixer],
```

Comment:
just went with the default `autoprefixer` settings (which i think attempts to cover browsers with ~12% market share) -- this only affects the dewey demos at go/dewey